lazy_tensor_core/lazy_tensor_core/csrc/view_ops/as_strided.h

```diff
@@ -1,14 +1,14 @@
 #pragma once
 
-#include <torch/csrc/lazy/ts_backend/ts_node.h>
-
 #include <vector>
 
+#include "lazy_tensor_core/csrc/view_ops/opcode.h"
+
 namespace torch_lazy_tensors {
 namespace ir {
 namespace ops {
 
-class AsStrided : public torch::lazy::TsNode {
+class AsStrided : public BaseNode {
```

Comment:
wdyt about adding a README to the view_ops folder?  I think at least 2 things would be good to explain in general:
 - how this 'BaseNode' works : haven't found it yet but I assume you typedef it somewhere and it is TsNode in our case...
 - what the pattern of {View} and {View}Reverse ops is about

Reply:
(could also be in a central view.h header or something, instead of readme...)

Reply:
Ok, let me add some description in the new `opcode.h`